Sources/Sentry/SentrySDK.m

```diff
@@ -465,6 +465,11 @@ + (void)installIntegrations
         [SentrySDK.currentHub getClient].options.integrations.mutableCopy;
 
     NSArray<Class> *defaultIntegrations = SentryOptions.defaultIntegrationClasses;
+
+    // We are going to use a dictionary that links the classes with their names
+    // so we can quickly check whether that class is in the option integrations collection.
+    // We cannot load the class itself with NSClassFromString because doing so may load a class
+    // that was duplicated in another module, leading to undefined behavior.
```

Comment:
> // We cannot load the class itself with NSClassFromString because doing so may load a class
    // that was duplicated in another module, leading to undefined behavior.
    
IMO this to be descriptive enough.

Reply:
Feel free to add a suggestion  ` ```suggestion ` for an alternative